### PR TITLE
Changing Signin consistently to SigninUser

### PR DESCRIPTION
**Pull request by chrish-21**
The actual name of the mutation is SigninUser Mutation, however currently there is a mix in the file/mutation names between SigninMutation and SigninUserMutation. Always naming it SigninUserMutation is less confusing and follows the standard file naming conventions.

Also changed some path names from apollo to relay.

### CODE REVIEW

File: content/frontend/react-relay/5-authentication.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ With that component in place, you can go and add a new route to your `react-rout
 
 Open `App.js` and update `render` to include the new route:
 
-```js{7}(path=".../hackernews-react-relay/src/components/App.js")
+```js{7}(path=".../hackernews-react-apollo/src/components/App.js")
```
Comment (marktani) [inline content/frontend/react-relay/5-authentication.md]: shouldn't this truly say `relay` as it is?

Comment (chrish-21) [inline content/frontend/react-relay/5-authentication.md]: yes absolutely, somehow messed that up with an old commit.
should be fixed in the new commit now

**Comment by marktani** [comment]
are you sure the mutation is called SignInUser?

**Comment by chrish-21** [comment]
Yes the mutation generated by the email/password auth provider is called "SigninUser" (lowercase 'i')

In the corresponding video he actually runs into a compiler error (https://youtu.be/f0Mf2YyGdqc?t=11m51s), because of the name missmatch and then changes the file name to SigninMutation.js.
The code works this way as well but for a consistent naming convention, he should have changed the mutation name to SigninUser instead.

**Comment by nikolasburk** [comment]
Thanks a lot for the PR, that's really helpful! I accidentally created this bug while writing the written part of the tutorial and then ran into it in the video as you pointed out but didn't get around to fix it in the written parts. So thanks a lot for this!! 🙏

**Comment by chrish-21** [comment]
Sure, glad I could help!

**Pull request merged**